central/sensor/service/connection/sensorevents.go

```diff
@@ -78,10 +78,6 @@ func stripTypePrefix(s string) string {
 	return s
 }
 
-func (s *sensorEventHandler) disableReconciliation() {
```

Comment:
Curious what this was used for originally and why it's no longer needed

Reply:
@fredrb Could you answer this question? The only usage was removed in https://github.com/stackrox/stackrox/commit/c7c0caf59a8c02ec202a0eec2737c737176062ab#diff-5eebef2d74ae81573c60ae033445bbc96d1d4fe325735839f47c945e6f5a1c97L636

Reply:
@RTann there was a bug in the original client reconciliation epic that was released in 4.3. We ended up changing the approach in #8430. If you're interested in the issue you can read the doc linked in the PR.